merlin-server/sql/merlin/tables/profile_segment.sql

```diff
@@ -4,6 +4,7 @@ create table profile_segment (
 
   start_offset interval not null,
   dynamics jsonb null,
+  is_gap bool not null default false,
```

Comment:
Following our other tables it'd be nice to have comments for each of these columns. [`profile.sql`](https://github.com/NASA-AMMOS/aerie/tree/develop/merlin-server/sql/merlin/tables/profile.sql) could be a decent example.

This would help devs/users understand what a null-able column may mean in the context of our data model.